### PR TITLE
Confluence: bs4 missing from setup.py

### PR DESCRIPTION
**Pull request by wael-orabi**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
     package_dir={"atlassian": "atlassian"},
     include_package_data=True,
     zip_safe=False,
-    install_requires=["deprecated", "requests", "six", "oauthlib", "requests_oauthlib", "jmespath"],
+    install_requires=["deprecated", "requests", "six", "oauthlib", "requests_oauthlib", "jmespath", "bs4"],
```
Comment (alexandre-fabra) [inline setup.py:28]: The PyPi package is called `beautifulsoup4`, isn't it?

```suggestion
    install_requires=["deprecated", "requests", "six", "oauthlib", "requests_oauthlib", "jmespath", "beautifulsoup4"],
```

**Review by alexandre-fabra — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (wael-orabi) [reply setup.py:28]: yeah, I thought it failed on tox with `beautifulsoup4` but it now it's working

Comment (ammarchalifah-bolt) [reply setup.py]: https://github.com/atlassian-api/atlassian-python-api/commit/450700f03bc7bd353d5e72f97a6717db98fd0717

I think `bs4` is the correct package name, while `beautifulsoup4` is the project name. I had an issue with the recent hotfix

**Review by ammarchalifah-bolt — APPROVED** [review]

**Review by alexandre-fabra — APPROVED** [review]

**Comment by twhalm** [comment]
Should this be marked as a bug since it is a regression introduced in the latest release?

**Review by RomuloSchiavonPL — APPROVED** [review]

**Comment by FCamborda** [comment]
can we expect a release with these changes soon @gonchik ? I think only a version bump is needed now, correct?

**Comment by gonchik** [comment]
Yeap, in 20-30 min I will publish it.
@FCamborda thanks for pinging me

**Pull request merged**